website/docs/installation/monitoring.md

```diff
@@ -8,6 +8,8 @@ authentik can be easily monitored in multiple ways.
 
 Configure your monitoring software to send requests to `/-/health/live/`, which will return a `HTTP 204` response as long as authentik is running. You can also send HTTP requests to `/-/health/ready/`, which will return `HTTP 204` if both PostgreSQL and Redis connections can be/have been established correctly.
 
+Updated in version ___: In previous versions, these endpoints returned an `HTTP 204` response.
```

Comment:
This should be added to the release notes and the docs changed here instead

Reply:
@BeryJu removed this line from the docs. I think you autogenerate changelog entries, so maybe we just need to update the PR title?

Would you be able to finish out this PR? Could use your help to get this over the line as I don't know what all is required (and I haven't been able to get a fully working dev env). I think regenerating the schema is also needed.